build.gradle

```diff
@@ -103,8 +102,9 @@ dependencies {
     api platform('com.exactpro.th2:bom:4.1.0')
 
     implementation "com.exactpro.th2:common:${commonVersion}"
-    implementation 'com.exactpro.th2:codec:4.7.4'
-    implementation 'com.exactpro.th2:sailfish-utils:3.13.0'
+    implementation 'com.exactpro.th2:codec:4.7.5'
+    implementation 'com.exactpro.th2:sailfish-utils:3.14.0'
```

Comment:
Try to migrate to sailfish-common

Reply:
Is there IMessage to Proto conversion in sailfish-common? Seems like there is no such functionality in common